lib/rules/dot-notation.js

```diff
@@ -53,7 +55,7 @@ module.exports = {
 
     create(context) {
         const options = context.options[0] || {};
-        const allowKeywords = options.allowKeywords === void 0 || !!options.allowKeywords;
+        const allowKeywords = options.allowKeywords === void 0 || options.allowKeywords;
```

Comment:
seems it could be written: `const allowKeywords = options.allowKeywords;`

I see it was unchanged, but it just caught my eye :(

Reply:
It can't be, because:
Supposes `options.allowKeywords` is `undefined`.
```js
options.allowKeywords  // falsy value
options.allowKeywords === void 0  // truthy value
```

Reply:
yes, you are right, I've misread it. ╰（‵□′）╯